### PR TITLE
Upgrade all dependencies and apply the operator template

### PR DESCRIPTION
**Pull request by siegfriedweber**
# Description

Upgrade all dependencies and apply the operator template

The pre-commit hooks changed a lot of line endings.

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] Helm chart can be installed and deployed operator works
- [x] Integration tests passed (for non trivial changes)
- [x] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: docs/modules/hbase/images/hbase_overview.drawio.svg
Comment (sbernauer) [inline docs/modules/hbase/images/hbase_overview.drawio.svg:1]: What was changed here? :thinking:

**Review by sbernauer — DISMISSED** [review]
Looks good

File: docs/modules/hbase/images/hbase_overview.drawio.svg
Comment (siegfriedweber) [reply docs/modules/hbase/images/hbase_overview.drawio.svg:1]: The newline at the end of the file.

**Comment by siegfriedweber** [comment]
https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hbase-operator-it-custom/145/

**Review by sbernauer — APPROVED** [review]

**Comment by sbernauer** [comment]
Tests failed installing Kerberos and MinIO, I would say that's more IONOS fault than this PR

**Pull request merged**